packages/playground/windows/playground-win32/Playground-Win32.cpp

```diff
@@ -153,7 +153,14 @@ struct WindowData {
     HWND interopHwnd;
     winrt::check_hresult(interop->get_WindowHandle(&interopHwnd));
 
-    MoveWindow(interopHwnd, 0, 0, windowPosition->cx, windowPosition->cy, TRUE);
+    constexpr int logBoxHeight = 100;
```

Comment:
Trying to understand this a bit more. If the root cause of the issue was not accounting for menu bar/title bar, why are we hardcoding this? Shouldn't we be able to measure this?

Reply:
Yes, we could measure this but this seems good enough for the test app. A similar adjustment needs to be made for the horizontal size for the scrollbar, that one we can't measure as easily since it's a xaml element not an hwnd.